src/helpers/library.ts

```diff
@@ -0,0 +1,206 @@
+import { AppSettings, GameInfo, GameStatus, InstallProgress } from 'src/types'
+import { IpcRenderer } from 'electron'
+import { TFunction } from 'react-i18next'
+import { getGameInfo, sendKill } from './index'
+
+const { ipcRenderer } = window.require('electron') as {
+  ipcRenderer: IpcRenderer
+}
+
+const storage: Storage = window.localStorage
+
+type InstallArgs = {
+  appName: string
+  handleGameStatus: (game: GameStatus) =>  Promise<void>
+  installPath: string
+  isInstalling: boolean
+  previousProgress: InstallProgress
+  progress: InstallProgress
+  setInstallPath?: (path: string) => void
+  t: TFunction<'gamepage'>
+}
+
+async function install({appName, installPath, t, progress, isInstalling, handleGameStatus, previousProgress, setInstallPath}: InstallArgs) {
+  const {folder_name, is_game, is_installed}: GameInfo = await getGameInfo(appName)
+  //Fix  Remove files when canceling (again)
```

Comment:
```suggestion
  // Fix removing files when canceling (again)
```
Sorry for being so pedantic.

Reply:
this was fixed, just forgot to remove the reminder